src/Cms/PageActions.php

```diff
@@ -533,7 +533,7 @@ public function createNum(int $num = null): int
                 $lang   = $this->kirby()->defaultLanguage() ?? null;
                 $field  = $this->content($lang)->get('date');
                 $date   = $field->isEmpty() ? 'now' : $field;
-                return date($format, strtotime($date));
+                return (int)date($format, strtotime($date));
```

Comment:
Will the [idate()](https://www.php.net/manual/en/function.idate.php) function work?

Reply:
That could be a breaking change, right? Cause I think currently the casting to int already happens through the return type hint of the method. Adding it in there only makes Psalm happier but doesn't change how it works. Throwing an error would - though it could still be a good idea, but then for 3.6

Reply:
@afbora Unfortunately not, it only supports a single-char format, not `Ymd` for example.

@distantnative It would be, right. I think we have two options: Either we leave the change out for now, or (my preferred solution) we change it with `(int)`, but add a TODO comment to it with a reference to this discussion.